include/serialport.h

```diff
@@ -49,6 +49,9 @@
 #include "hardware.h"
 #endif
 
+#define SERIAL_MIN_BAUDRATE 300u
+#define SERIAL_MAX_BAUDRATE 115200u
```

Comment:
Suggest `constexpr uint32_t` and CamelCase, like other newly added constants.

Making them real variables allow their type to be assessed and also show up in debuggers by name (instead of literal numbers).